### ISSUE TITLE
Should we drop support for Symfony 3 and 4?

### ISSUE THREAD

**Issue by tarlepp** [mark-gerarts/automapper-plus-bundle#30]
Currently this bundle supports Symfony 3 and 4 - https://github.com/mark-gerarts/automapper-plus-bundle/blob/master/composer.json#L20 - which are no longer supported - https://symfony.com/releases - so should we drop those from this bundle too?

**Comment by mark-gerarts** [comment]
Yes that's a good idea! Feel free to open a PR, otherwise I'll look into this after some time.

**Comment by tarlepp** [comment]
> Yes that's a good idea! Feel free to open a PR, otherwise I'll look into this after some time.

Seems like it take "some" time to do this, but now there is open pull request about this :D

**Comment by mark-gerarts** [comment]
Yeah sorry the project is low priority for me, I've been out of the PHP world for quite some years already and am very busy with other things at the moment. So AMP things get pushed further back that I'd like.

Anyhows definitely agree support should be dropped (those versions are ancient); I merged the PR. Thanks!
